proto/funding_service.proto

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) 2023 - for information on the respective copyright owner
+// see the NOTICE file and/or the repository at
+// https://github.com/hyperledger-labs/perun-node
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+// Package pb contains proto3 definitions for user API and the corresponding
+// generated code for grpc server and client.
+package pb;
```

Comment:
Does it make sense to rename the package name to something like perun-node-api, node-api or similar?
As this package name will be taken by frameworks to generate clients, the corresponding name will be taken for the package/module name. grpc is the technical framework and not the logical name.
This corresponds also to the other files in the folder.

Reply:
Yeah, we could rename this package since it will be used by external projects.

I tried doing it now, but looks like it needs a lot of rebasing work (as I have 2 other PRs coming after this also prepared).

So, my suggestion would be we could do this change as a separate task, after finishing the next two PRs.

Reply:
ok fine